robotpy_build/platforms.py

```diff
@@ -36,8 +37,7 @@ def __post_init__(self):
     # TODO: linuxraspbian
     "win32": WPILibMavenPlatform("x86", "windows", "", ".dll", ".lib"),
     "win-amd64": WPILibMavenPlatform(X86_64, "windows", "", ".dll", ".lib"),
-    # TODO: need to filter this value
-    "macosx-10.9-x86_64": WPILibMavenPlatform(X86_64, "osx", libext=".dylib"),
+    "osx": WPILibMavenPlatform(X86_64, "osx", libext=".dylib")
```

Comment:
Any particular reason this is still in the dict? Does `distutils.util.get_platform()` ever return `"osx"`?

Reply:
I don't think `...get_platform()` ever does return "osx". I just left it there so all the platform -> MavenPlatform mappings were in one place.